1903_Samples/CppWinRT_Win32_App/SampleCppApp.Package/Images/dummy.txt

```diff
@@ -0,0 +1,2 @@
+
+dummy
```

Comment:
Why do you need a text file into the image folder?

Reply:
There will be a build break in the WAP project if this is not pressent